client/android/app/build.gradle

```diff
@@ -16,6 +16,8 @@ if (flutterRoot == null) {
 def flutterVersionCode = localProperties.getProperty('flutter.versionCode')
 def flutterVersionName = localProperties.getProperty('flutter.versionName')
 
+def app = "app"
```

Comment:
remove line

Reply:
kept line - updated it to be server instead